src/Abstractions/NexusMods.Abstractions.Games/RunGameTool.cs

```diff
@@ -60,10 +61,12 @@ public RunGameTool(IServiceProvider serviceProvider, T game)
     public string Name => $"Run {_game.Name}";
 
     /// <inheritdoc />
-    public async Task Execute(Loadout loadout, CancellationToken cancellationToken)
+    public async Task Execute(Loadout.Model loadout, CancellationToken cancellationToken)
     {
         _logger.LogInformation("Starting {Name}", Name);
+        throw new NotImplementedException();
```

Comment:
I'm guessing this is a TODO?

Reply:
Attempting to run the game throws this so currently it doesn't work

Reply:
Yep...I thought I missed this. I'll get it fixed up